assets/startup.sh

```diff
@@ -38,29 +38,30 @@ if [ -z "${LOG_LEVEL}" ]; then
   LOG_LEVEL=0
 else
   # confirm log level is between 0-3, since these are the only log levels supported
-  if [ "${LOG_LEVEL}" -gt 3 ]; then
+  if [[ $LOG_LEVEL != [0123] ]]; then
     # level outside of supported range, let's set to default (0)
     LOG_LEVEL=0
   fi
 fi
 
-IFS=","
+IFS=", "
```

Comment:
Let's not change this, see the DEFAULT_NTP variable above, which will also populate this. Existing users of this container (including myself) will be using it with lists of servers that are separated only by comma, not comma+space, so it would break existing setups as well.